upgrade-cockroach.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Upgrade Your Cluster's Version of CockroachDB
+summary: Learn how to upgrade your CockroachDB cluster to a new version.
+toc: false
+toc_not_nested: true
+---
+
+Because of CockroachDB's multi-active availability design, you can perform a "rolling upgrade" of CockroachDB on your cluster. This means you can upgrade individual nodes in your cluster one at a time without bringing your cluster down at all.
+
+For rolling upgrades to work, though, your cluster must be behind a load balancer. If your application communicates with a single node, bringing it down to upgrade its CockroachDB binary, your application will not receive data, causing it to fail.
+
+## Manually Perform a Rolling Upgrade
+
+In advance of this procedure, you will need the IP addresses of all your cluster's nodes.
+
+1. Connect to the node.
+
+2. Stop `cockroach`:
+
+   ~~~ shell
+   $ cockroach quit [flags]
+   ~~~
+
+   `[flags]` includes the flags you [use to communicate with a node](stop-a-node.html), such as its `--host`.
+
+3. Rename the outdated `cockroach` binary:
+
+   ~~~ shell
+   $ i="$(which cockroach)"; mv "$i" "$i"_old
+   ~~~
+
+4. Download and install the CockroachDB binary you want to use (this example uses the `latest` build for simplicity's sake):
```

Comment:
I think we're planning to discontinue the `latest` tarballs; this should use an explicit version. (that was going to be less of a headache when we just had the one install page that would need to be updated)